website/docs/language/values/variables.mdx

```diff
@@ -532,14 +532,14 @@ variable cannot be assigned multiple values within a single source.
 Terraform loads variables in the following order, with later sources taking
 precedence over earlier ones:
 
-* Environment variables
+* Environment variables (Lowest Precedence)
 * The `terraform.tfvars` file, if present.
 * The `terraform.tfvars.json` file, if present.
 * Any `*.auto.tfvars` or `*.auto.tfvars.json` files, processed in lexical order
   of their filenames.
 * Any `-var` and `-var-file` options on the command line, in the order they
```

Comment:
Hi! Thank you for this change :-) My suggestion is - can we remove the parentheses around "This includes variables set by a Terraform Cloud workspace."  The two sets of parentheses in a row is a bit overwhelming visually.  If you can make that change, I'd be happy to merge this for you!